Cable-Crimping/README.adoc

```diff
@@ -0,0 +1,51 @@
+= How to Crimp RJ45 cables
+
+== Strip the wire
+Make a shallow cut with a knife and run it around the cable 2.5 to 5 cm away from the end of the cable.
+The jacket should slide off easily.
+Be very careful when cutting away the sheathe, it is very easy to cut the wire inside by mistake.
+
+There will be 4 pairs of twisted wires that should now be exposed:
+
+* Solid orange and half orange, half white
+* Solid green and half green, half white
+* Solid blue and half blue, half white
+* Solid brown and half brown, half white.
+
+== Removing the core
+Folding back each of the wires will expose a core of the cable.
+Cut it off at the point that the cable was stripped to.
+This part can be discarded.
+
+== Preparing to crimp
+
+Straighten the wires, this can easily be done with a pair of tweezers.
+Arrange them from right to left in the following order:
+
+* Orange/white stripe
+* Orange
+* Green/white stripe
+* Blue
+* Blue/white stripe
+* Green
+* Brown/white stripe
+* Brown
+
+Trim the untwisted wires to a good length.
+The insulation of the cable should be just inside the bottom of the RJ-45 connector.
+Note to orient the cables so that the Orange/white striped wire is on the right when looking at the plastic release latch on the top.
```

Comment:
Use the note syntax.

`NOTE: To orient the cables...`

This is actual separate adoc syntax, not just a style thing.